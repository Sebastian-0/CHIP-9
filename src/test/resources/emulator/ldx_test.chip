sout BC
sout DE
sout HL
sout SP
ldx bc, 1000
ldx de, 1001
ldx hl, 1002
ldx sp, 1003
sout BC
sout DE
sout HL
sout SP
ldx bc, 65535
ldx de, 65535
ldx hl, 65535
ldx sp, 65535
sout BC
sout DE
sout HL
sout SP

# Test register partitioning
ldx bc, 0x0102
sout B
sout C
hcf