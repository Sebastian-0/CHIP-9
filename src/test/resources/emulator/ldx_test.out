0
0
0
0
1000
1001
1002
1003
65535
65535
65535
65535
1
2